tests/binning_test.py

```diff
@@ -231,6 +231,17 @@ def test_bin_integer_coord_by_fractional_stepsize_raises(dtype):
         table.bin(label=sc.scalar(0.5, unit='m'))
 
 
+def test_bin_with_automatic_bin_bounds_raises_if_no_events():
+    table = sc.data.table_xyz(0)
+    with pytest.raises(ValueError):
+        table.bin(x=4)
+
+
+def test_bin_with_manual_bin_bounds_not_raises_if_no_events():
```

Comment:
Now I wonder: If you give bad manual bounds (similar to the ones computed automatically with no events, e.g., not increasing), do we also get a weird error?

Reply:
Yes, good 👍